### ISSUE TITLE
View switcher disappears on narrow window sizes

### ISSUE THREAD

**Issue by jannuary** [wwmm/easyeffects#1353]
### EasyEffects Version

6.2.1

### What package are you using?

Flatpak (Flathub)

### Distribution

Fedora Rawhide

### Describe the bug

Resizing the window to a certain size makes the Input/Output/Pipewire view switcher disappear:

![image](https://user-images.githubusercontent.com/27908024/150323940-4a245cea-f15c-491f-88d6-c6f58e418fc8.png)

### Expected Behavior

The view switcher doesn't disappear when resizing. Probably not as simple though - there's already a headerbar where the view switcher would go otherwise.

**Comment by wwmm** [comment]
>The view switcher doesn't disappear when resizing. Probably not as simple though - there's already a headerbar where the view switcher would go otherwise.

Yes. It is a good thing what libadwaita is trying to do when the window width gets too small. But the problem is assuming that the application bottom is free to receive what was in the title bar. Making what we have there harder to find just to follow what is usually show cased in libadwaita examples does not seem right.

**Comment by wwmm** [comment]
Now that we moved to Qt debugging this issue does not make sense. Let's close it.
